### PR TITLE
feature/138 - open discovery in new tab

### LINKED ISSUE #138: Open new tab when clicking on 'read more' button
Feedback from client: 

discoveries landing page: it doesn't allow me to open the contents in a new tab with control + click on the "read more" button -- sometimes I find this feature useful to compare different reports addressing similar topics so this might be something to consider

### PR DESCRIPTION
**Pull request by alegzandach**
closes #138

### CODE REVIEW

File: themes/base-hugo-theme/layouts/partials/explorations-card.html [hunk 1]
```diff
@@ -3,16 +3,16 @@
     {{ with $.Site.GetPage "page" "uploads/_index.md" }}
         {{ with .Resources.GetMatch $img }}
               {{$imgcrop := .Fill "118x112 Center" */}}
-              <a href="{{ .Permalink }}" aria-label="Go to {{ .Params.displaytitle }}" class="gta-event-postSelected d-flex justify-content-center d-md-block"><img class="img-thumbnail" src="{{ .Params.image }}" width="" height="" alt="Article thumbnail"></a>
+              <a href="{{ .Permalink }}" target="_blank" aria-label="Go to {{ .Params.displaytitle }}" class="gta-event-postSelected d-flex justify-content-center d-md-block"><img class="img-thumbnail" src="{{ .Params.image }}" width="" height="" alt="Article thumbnail"></a>
```
Comment (lougroshek) [inline themes/base-hugo-theme/layouts/partials/explorations-card.html:6]: Generally where we add `target="_blank"` we add `rel="noopener noreferrer"` as well. https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

**Comment by lougroshek** [comment]
@alegzandach I think maybe we should take a different approach here. You can add `target="_blank"` to some of these links in the page, but the real issue is that the click event is being intercepted by the google tracking feature, and then when it is called again it is called differently. This is why a CMD+click on the header in the explorer card opens a new tag, but the CMD+click on the thumbnail or the button doesn't. They have the `gta-event-CTAClick` class that means that the google event tracking script is listening for interactions on that element. And if you open a new tab, you are interfering with the chain of interactions there.

**Comment by laurendoone** [comment]
@alegzandach this is a low priority enhancement so we can defer this to post feb 3 launch if you don't have time to look into it before

**Comment by lougroshek** [comment]
I've implemented `feature/138-alt` to resolve this. Closing.

**Pull request closed without merging**